scripts/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:bionic
+
```

Comment:
Why don't you copy the helper scripts to the docker image (for example to `/usr/local/bin/`) by using `COPY` command? This way if you run this image you can easily use these scripts.
```bash
docker run \
  --rm \
  --volume $CODECOMPASS_ROOT:/CodeCompass \
  codecompass/codecompass-dev bash codecompass-build.sh
```